src/Nox.Types/Types/PhoneNumber/PhoneNumber.cs

```diff
@@ -1,9 +1,28 @@
+using System.Text.RegularExpressions;
+using System;
+
 namespace Nox.Types;
 
+/// <summary>
+/// Represents a Nox <see cref="PhoneNumber"/> type and value object.
+/// </summary>
+public sealed class PhoneNumber : ValueObject<string, PhoneNumber>
```

Comment:
SimpleTypes require a converter, and alo add a test in EF just to see it working

Reply:
Missing converter has been added.